src/base/bittorrent/session.cpp

```diff
@@ -290,6 +290,9 @@ Session::Session(QObject *parent)
     , m_maxActiveUploads(BITTORRENT_SESSION_KEY("MaxActiveUploads"), 3, lowerLimited(-1))
     , m_maxActiveTorrents(BITTORRENT_SESSION_KEY("MaxActiveTorrents"), 5, lowerLimited(-1))
     , m_ignoreSlowTorrentsForQueueing(BITTORRENT_SESSION_KEY("IgnoreSlowTorrentsForQueueing"), false)
+    , m_downRateForSlowTorrents(BITTORRENT_SESSION_KEY("DownRateForSlowTorrents"), 2)
```

Comment:
I would not shrink it. `m_downloadRateForSlowTorrents(BITTORRENT_SESSION_KEY("DownloadRateForSlowTorrents"), 2)`
The same for *Upload*.